lms/static/sass/elements/_system-feedback.scss

```diff
@@ -145,6 +145,13 @@
   }
 }
 
+.wrapper-msg {
```

Comment:
@marcotuts I wanted to have similar message banner that stays on the UI all the time and that looks similar to the existing one. 

Should we add fixed postfix with other rules as well to avoid the errors and make same copy of new rules but with different selectors that includes fixed ???